pkg/asset/installconfig/aws/validation.go

```diff
@@ -537,3 +540,93 @@ func isHostedZoneAssociatedWithVPC(hostedZone *route53.GetHostedZoneOutput, vpcI
+	// MTU is higher than general supported.
+	// https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html#considerations
+	// https://docs.aws.amazon.com/wavelength/latest/developerguide/how-wavelengths-work.html
+	higherLimitMTUEdge := uint32(1300)
+
+	// MTU overhead for OVNKubernetes and OpenShiftSDN network plugins.
+	// https://docs.openshift.com/container-platform/4.14/networking/changing-cluster-network-mtu.html#mtu-value-selection_changing-cluster-network-mtu
+	minOverheadOVN := uint32(100)
+	minOverheadSDN := uint32(50)
+
+	// Calculating MTU limits considering the base overhead added for each network plugin.
+	limitEdgeOVNKubernetes := higherLimitMTUEdge - minOverheadOVN
+	limitEdgeOpenShiftSDN := higherLimitMTUEdge - minOverheadSDN
+	limitOVNKubernetes := higherLimitMTUVPC - minOverheadOVN
+	limitOpenShiftSDN := higherLimitMTUVPC - minOverheadSDN
+
+	// MTU should not be higher than the allowed MTU for VPC.
+	// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/network_mtu.html
+	if mtu > higherLimitMTUVPC {
+		return append(allErrs, field.Invalid(fldPath, int(mtu), fmt.Sprintf("cluster network MTU does not meet the maximum value by VPC of %d", higherLimitMTUVPC)))
+	}
+
+	// There is no official test with lower than 1kB, but testing MTU with 1100 decreased the network
+	// performance in 70%, setting the lower limit to protect from too lower values.
+	// https://issues.redhat.com/browse/OCPBUGS-11098
+	if mtu < lowerLimitMTUVPC {
+		return append(allErrs, field.Invalid(fldPath, int(mtu), fmt.Sprintf("cluster network MTU does not meet the minimum value of %d", lowerLimitMTUVPC)))
+	}
+
+	hasEdgePool := false
+	warnEdgePool := false
+	for _, compute := range config.Compute {
+		if compute.Name == types.MachinePoolEdgeRoleName {
+			hasEdgePool = true
+			break
+		}
+	}
+
+	switch network {
+	case string(operv1.NetworkTypeOVNKubernetes):
+		if mtu > limitOVNKubernetes {
+			return append(allErrs, field.Invalid(fldPath, int(mtu), fmt.Sprintf("cluster network MTU does not meet the maximum value by the network plugin %s of %d", network, limitOVNKubernetes)))
+		}
+		if hasEdgePool && (mtu > limitEdgeOVNKubernetes) {
+			warnEdgePool = true
+		}
+
+	case string(operv1.NetworkTypeOpenShiftSDN):
+		if mtu > limitOpenShiftSDN {
+			return append(allErrs, field.Invalid(fldPath, int(mtu), fmt.Sprintf("cluster network MTU does not meet the maximum value by the network plugin %s of %d", network, limitOpenShiftSDN)))
+		}
+		if hasEdgePool && (mtu > limitEdgeOpenShiftSDN) {
+			warnEdgePool = true
+		}
+	default:
+		return append(allErrs, field.Invalid(fldPath, int(mtu), fmt.Sprintf("custom MTU is not valid with network plugin %s", network)))
+	}
+	if warnEdgePool {
+		logrus.Warnf("platform.aws.ClusterNetworkMTU is higher than generally supported edge zones. Please make sure all zones defined in the edge compute pool supports the MTU %d bytes with network plugin %s.", mtu, network)
```

Comment:
@zaneb , what about this?:
> platform.aws.ClusterNetworkMTU exceeds the maximum value generally supported by AWS Local or Wavelength zones. Please ensure all AWS Zones defined in the edge compute pool supports the MTU %d bytes between nodes (EC2) in the zone and the Region, otherwise the lower supported MTU must be set.

 Trying to ensure the "supported" wording is from AWS. The intention is to provide some guidance how to fix it by setting the lower number when the condition meet.

Reply:
Changed! Thanks